frontend/src/pages/Social/Profile/Profile.css

```
@@ -108,6 +108,11 @@
   color: hsl(var(--muted-foreground));
   line-height: 1.6;
   font-size: 0.95rem;
+  word-wrap: break-word;
+  overflow-wrap: break-word;
+  word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)